circle.yml

```diff
@@ -18,7 +18,7 @@ test:
     - pip install coala-bears colorama==0.3.6 --pre -U:
```

Comment:
OK. Should not have missed that :)

So 'Fixes' is only used with issues that are labelled as bugs?